spec/03-plugins/26-prometheus/02-access_spec.lua

```diff
@@ -826,8 +828,9 @@ describe("Plugin: prometheus (access) AI metrics", function()
         path    = "/metrics",
```

Comment:
The comment in the original PR is missing here. Is this expected?
https://github.com/Kong/kong-ee/pull/12736/files#diff-3b4bb6e1cacfc04dcd8ad10df033c3f46170f182f6a77ff935bf4105a6485475R834

Reply:
Yes, it is expected. `consumer` is added in another PR which was decided not merged back to CE, so for CE and 3.10, we don't have the `consumer` field for AI metrics.

Reply:
@ProBrian how about the comment?

Reply:
nice pick, resloved.